content/markdown/concepts/headings/headings.md

```diff
@@ -35,7 +35,15 @@ The number of number signs you use should correspond to the heading level. For e
 ###### Heading level 6
 ```
 
-![example](https://i.imgur.com/8NNe3NE.png)
+<details>
```

Comment:
```suggestion
<details open>
```

I prefer if the `<details>` were already open upon page load (and then the user can collapse/expand if they want).